packages/themes/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@hig/themes",
+  "version": "0.1.0-alpha",
+  "description": "HIG theme definitions and supporting components",
+  "author": "Autodesk Inc.",
+  "license": "Apache-2.0",
+  "homepage": "https://hig.autodesk.com",
+  "main": "build/index.js",
+  "module": "build/index.es.js",
+  "dependencies": {
+    "@hig/styles": "^0.1.0-alpha",
```

Comment:
This should be in `devDependencies`.

Reply:
I wish I could turn you into an eslint plugin.